### ISSUE TITLE
Docker conflict with fitnesse Wiki

### ISSUE THREAD

**Issue by rmanibus** [unclebob/fitnesse#1304]
**Describe the bug**
Running a docker command (docker run, docker network create) break fitnesse wiki connection if it is running on the same host (outside docker).

**To Reproduce**
1. Run a simple but long fitnesse test (just run test&remote_debug to make it hang for a while)
2. Run a docker container from cli (or from within the test with the java sdk ...)
3. The wiki page stop loading. after a while, a broken pipe error shows up in the wiki terminal

**Stack Trace**
If applicable, add a stack trace of the error.

**Configuration:**
 - OS: centos 7 (redhat)
 - Firefox 82 (also tested on 68)
 - Version 20200501 of FitNesse
 - Test System: Slim

**Additional context**
This does not seems to be a conflict with the docker sdk as it also happens when using docker cli outside of the test.
Also I tried to open a netcat connection on the same host and create a container while it was open and it does not broke it.

**Comment by fhoeben** [comment]
What container are you starting, which ports does it expose? Or doesn't that matter (e.g. just running `docker run hello-world` already causes the problem)?

**Comment by rmanibus** [comment]
yes any container, I just ran docker run --rm -it alpine /bin/sh, any network (without ip colision ...) reproduced on two computers (maybe worth mentioning we are working in citrix vm)

**Comment by rmanibus** [comment]
I just tried with the hello world, it's also the case

**Comment by fhoeben** [comment]
Very strange, no idea what would be causing this.

Do you run Firefox on the same machine as FitNesse? Does a remote client experience the same problem?
Does the same problem occur when FitNesse is run in a container also?

**Comment by rmanibus** [comment]
yes Firefox is on the same machine as FitNesse. I had no luck to try a remote client for now.
I also tried to run fitnesse in a container and connecting from the same machine and I had the same result.

**Comment by rmanibus** [comment]
I was able to reproduce this on a blank ubuntu VM.

**Comment by rmanibus** [comment]
Do someone has an idea about what is causing this ? I can help fix it but I have zero knowledge about fitnesse frontend !

**Comment by tcnh** [comment]
I cannot reproduce this on Windows...

Steps taken:
* run vanilla fitnesse
* start the acceptance test suite (takes quite a while)
* run a docker container

No irregular behaviour was observed.

Also tried with a test in remote debug mode, same result: test doesn't abort.

Could it be related to (or have the same root cause as) https://github.com/unclebob/fitnesse/issues/1283? That also seems to happen on linux only and also results in a 'broken pipe' error

**Comment by rmanibus** [comment]
Well, docker on Windows run in hyperV so I would expect the Daemon to be isolated on a network point of view.

**Comment by tcnh** [comment]
Could be something along those lines.. I'm using the WSL2 engine, not hyper-v.

Also tried reproducing using the following setup:
- Fitnesse from wsl Ubuntu shell 
- Docker command from that same shell
- Firefox on host machine

Also unable to reproduce. 
Don't get me wrong, I'm pretty sure that something in FitNesse's wiki server causes this behaviour (and I've been previously digging into it for quite a few hours with nothing to show for it), but I'm not so sure it's actually specific to Docker. However the docker-trigger may be worth investigating as it's another situation with similar outcome. 
See also: https://github.com/unclebob/fitnesse/issues/1187 where something similar seemed to occur at semi-random. I could reproduce that back then, but only with Debian 8 and FF. A fix then broke IE11 compatibility and search, so it was reverted and replaced with changes by @fhoeben later on.

**Comment by rmanibus** [comment]
Ok I just try on Chrome and it is working so this is also Firefox only.
